app/views/spaces/edit/common/_edit_rich_text.html.erb

```diff
@@ -7,19 +7,19 @@
     tag.h3 t("space_show.only_for_space", title: @space.title) if SpaceOwner.method_defined? field
   %>
 
-  <%= form.rich_text_area field %>
+  <%= form.input field, as: :rich_text_area %>
 </div>
 
 <%=
   ## For when SpaceOwner has the same field, show the field:
   if SpaceOwner.method_defined? field
-    form.fields_for :space_owner do |space_owner_form| %>
+    form.fields_for @space.space_owner do |space_owner_form| %>
```

Comment:
Denne medfører en bug som gør at man lager en ny space owner hver gang vi submitter et form. Tror den bør være som den var. 

Men ser vi også har en annen bug på linje 57 i spaces_controller, som kanskje er den du forsøkte fikse:

```ruby
      space_owner: SpaceOwner.find_or_create_by!(title: params[:space][:space_owner_title]),
```

Den linjen der, som jeg tror kom med @mathias234 sin branch når vi la til TomSelect på SpaceOwner (så den er også i main!), dreper av en eller annen grunn muligheten for å sende med space_owner nested attributes, og gjør at vi får 5xx og død server når vi submitter innhold. Kommenterer jeg den ut, og endrer tilbake til `:space_owner` så fungerer redigering både i denne branchen og i main.

Usikker på hva som er rett måte å løse det på uten samtidig å drepe @mathias234  sin feature på TomSelect - men vi må i alle fall få det fikset! Om det skjer i denne branchen, eller i en egen (siden det tross alt er en bug på main - og ikke har noe med simple forms å gjøre) vet jeg heller ikke hva som er best, kommer an på hvor fort vi får inn denne i main!

Reply:
Kan nå merge inn main for å få fiksen